PostBoard.Api/Controllers/PingController.cs

```diff
@@ -9,6 +9,6 @@ public class PingController : ControllerBase
     [HttpGet]
     public IActionResult Ping()
```

Comment:
Update `public IActionResult Ping()` to `public Task<IActionResult> PingAsync()`.